packages/navigator/src/browser/navigator-contribution.ts

```diff
@@ -105,7 +113,8 @@ export namespace FileNavigatorCommands {
         label: 'Focus on Files Explorer'
     };
     export const COPY_RELATIVE_FILE_PATH: Command = {
-        id: 'navigator.copyRelativeFilePath'
+        id: 'navigator.copyRelativeFilePath',
+        label: 'Copy Relative Path'
```

Comment:
Should we align?

```suggestion
        label: 'Copy Relative Path of Active File'
```

Reply:
In VSCode, when I right click on files and look for this command I see "Copy Relative Path". I do see two labels for this command in the VSCode repo, but I'm not sure I understand how VSCode represents their commands to make the call:

https://github.com/microsoft/vscode/blob/2853fd0ffcc4c067bf7b5b31ccfc15ac245fc89c/src/vs/workbench/contrib/files/browser/fileActions.contribution.ts#L163

https://github.com/microsoft/vscode/blob/2853fd0ffcc4c067bf7b5b31ccfc15ac245fc89c/src/vs/workbench/contrib/files/browser/fileActions.contribution.ts#L215